app/views/foods/index.html.erb

```diff
@@ -0,0 +1,13 @@
+<% if @search_result_count >= 10000 %>
+  <h3>Number of Results: <%= @search_result_count %> +</h3>
+<% else %>
+  <h3>Number of Results: <%= @search_result_count %></h3>
+<% end %>
+<br>
+<h3>Results: <% @foods.take(10).each do |food| %></h3>
```

Comment:
It would be much better practice to only create the number of food objects we plan to use in the facade. You could have passed a `pageSize` param in your get request to limit the number of results to 10 in your service.